### ISSUE TITLE
pub_test

### ISSUE THREAD

**Issue by L1bertad** [exiawsh/StreamPETR#240]
Hello author, can pub_test.py use the test set?

**Comment by exiawsh** [comment]
Yes, you can. You can obtain the submission json file. And submit the json file to the nuscenes eval ai server.
